Scala/Symbols-def.tmPreferences

```diff
@@ -9,10 +9,6 @@
   <dict>
     <key>showInSymbolList</key>
     <integer>1</integer>
-    <key>symbolTransformation</key>
-    <string>
-      s/(.+)/  def $1\(...\)/
-    </string>
```

Comment:
This file should also be able to be removed since there is no symbol transform any longer and `entity.name.function` should be part of the symbol list by default.